tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -9,19 +9,16 @@ MK_DIR := $(dir $(MK_PATH))
 # Verbose build
 V := 1
 
-define BUILD
-	$(MK_DIR)/kata-deploy-binaries-in-docker.sh $(if $(V),,-s) --build=$1
-endef
+ifeq ($(CROSS_BUILD),)
+	CROSS_BUILD = false
+endif
 
-kata-tarball: | all-parallel merge-builds
+ifeq ($(CROSS_BUILD),false)
+	ARCH := $(shell uname -m)
```

Comment:
@BbolroC suppose that I want to build the SE rootfs on my x86_64 laptop. I export `CROSS_BUILD=true` and then should I export `ARCH=s390x`?

I didn't see in details the upcoming commits, but, should I also export `TARGET_ARCH=s390x` too?

Reply:
The following three are required:

```
CROSS_BUILD=true TARGET_ARCH=s390x ARCH=s390x
```

This is something we agreed on at https://github.com/kata-containers/kata-containers/pull/6586